### ISSUE TITLE
Extra Effects: Blindness + Deafness

### ISSUE THREAD

**Issue by mikenrafter** [thedocruby/resounding#84]
To quote @thedocruby in vlad2305m/Sound-Physics-Fabric:
> A cool feature would be to significantly dampen all sounds when the player has the blindness effect. Could be a cool way to add more immersion. Also, conduit power and/or water breathing could remove or reduce the water filter.

Cool effect, but a counter-argument: this could make the game significantly harder to play, especially with the addition of the deep-dark. So it should be with the cosmetic/unrealistic/extra effects.

**Comment by ryleu** [comment]
Agreed. I’d love to see this feature as something optional I can enforce in my modpack.
